DependencyInjection/LexikTranslationExtension.php

```diff
@@ -85,6 +85,14 @@ public function buildTranslatorDefinition(ContainerBuilder $container)
                 new Parameter('lexik_translation.translator.options')
             ];
             $translator->setPublic(true);
+        } elseif (Kernel::VERSION_ID >= 30400) {
+            $arguments = [
```

Comment:
Arguments duplicate like for upper condition Kernel::VERSION_ID >= 40000.

Maybe you change 
```
 if (Kernel::VERSION_ID >= 40000) {
```
to 
```
 if (Kernel::VERSION_ID >= 30400) {
```